apps/studio/components/interfaces/Realtime/Inspector/useRealtimeMessages.ts

```diff
@@ -170,6 +176,14 @@ export const useRealtimeMessages = ({
         }
       } else if (status === 'CLOSED') {
         // console.log(`Realtime Channel status: ${status}`)
+      } else if (status === 'CHANNEL_ERROR') {
+        toast.error(
+          'Failed to connect to the channel. This may be due to restrictive RLS policies. Check your role and try again.'
```

Comment:
tinnyyyyyyy nit 

```suggestion
          'Failed to connect to the channel: This may be due to restrictive RLS policies. Check your role and try again.'
```